src/texttospeechbar.cpp

```diff
@@ -54,6 +58,16 @@ void TextToSpeechBar::setLocale(const QLocale& locale)
     }
 }
 
+void TextToSpeechBar::setupSpeedOptionsComboBox()
+{
+    ui->speedLabel->setText(gt("speed"));
```

Comment:
I think you forgot to include the change to `resources/i18n/en.json`. Please also don't forget to update `resources/i18n/qqq.json` too.